torch/onnx/_internal/fx/diagnostics.py

```diff
@@ -15,7 +15,12 @@
 from torch.onnx._internal.diagnostics import infra
 from torch.onnx._internal.diagnostics.infra import decorator, formatter, utils
 
-_LENGTH_LIMIT: int = 89
+from torch.onnx._internal.fx import type_utils as fx_type_utils
+
+# NOTE: Symbolic shapes could be a calculation of values, such as
+# Tensor(i64[s0, 64, (s1//2) - 2, (s1//2) - 2]) where s0 and s1 are symbolic
+# so we need to relax the length limit.
+_LENGTH_LIMIT: int = 120
```

Comment:
Why do we need a limit? It seems this is just a check inside the formatter, without any actual consequence to the model being exported. In that case, this seems an unnecessary limit.

If we really do, how do we know we need to increase this length? Any actual error or just a diagnostic log? 

Having these info in a comment would be a great source of knowledge for future debug

Reply:
It's actually one of SARIF rule generating warnings. And I found that the dynamic shapes usually hits at least 100. It's all about formatting, so I think as long as the report looks good, the limit can be adjusted.